### PR TITLE
Update ToolDistillator from 0.9 to 0.9.1

### PR DESCRIPTION
**Pull request by clsiguret**
FOR CONTRIBUTOR:
* [x] I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] License permits unrestricted use (educational + commercial)
* [ ] This PR adds a new tool or tool collection
* [x] This PR updates an existing tool or tool collection
* [ ] This PR does something else (explain below)

Update to new version: 0.9 → **0.9.1**
The wrapper does not change, there was just a new [release](https://gitlab.com/ifb-elixirfr/abromics/tooldistillator/-/releases/0.9.1 
) to fix some minor errors.

Thanks :smiley:

### CODE REVIEW

File: tools/tooldistillator/test-data/summarize/bakta_output.json
Comment (bgruening) [inline tools/tooldistillator/test-data/summarize/bakta_output.json:1]: is this file needed?

File: tools/tooldistillator/test-data/summarize/abricate_output.json
Comment (bgruening) [inline tools/tooldistillator/test-data/summarize/abricate_output.json:1]: Why do those files here need an update? They should be stable, isn't it?

**Review by bgruening — COMMENTED** [review]
All the files in

**Review by bgruening — APPROVED** [review]

**Comment by clsiguret** [comment]
Thanks @bgruening :smiley:
